docs/team/r-ramana.md

```diff
@@ -52,39 +52,40 @@ NotUS is a quick and simple, Command Line Interface (CLI) based, note-taking app
-1. Refactored code to abstract out the successful/unsuccessful execution messages from the individual command classes to its own `CommandMessage` class.
+1. Added ASCII art (PR [#90](https://github.com/AY2021S1-CS2113-T13-1/tp/pull/90) and [#80](https://github.com/AY2021S1-CS2113-T13-1/tp/pull/80)) to make the application more user-friendly. 
+1. Refactored code to abstract out the successful/unsuccessful execution messages (PR [#143](https://github.com/AY2021S1-CS2113-T13-1/tp/pull/143)) from the individual command classes to its own `CommandMessage` class.
 
 ### Code contributed
-[Reposense Link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=r-ramana&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
+[RepoSense Link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=r-ramana&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other) to view a report of the code contribution.
 
 ### Contributions to Team Based Task
 1. Setting up the GitHub team organization and repository.
-1. Maintained Issues and Reviewed PR alongside Jun Xiang.
+1. Maintained Issues and Reviewed almost all PRs alongside Jun Xiang.
 1. Updating of User/Developer documents that are not feature specific.
-1. Maintained Gantt Chart.
-1. Created a Development Practices Document.
+1. Set up [PERT Chart](https://github.com/AY2021S1-CS2113-T13-1/tp/blob/master/docs/DevelopmentPractices.md#pert).
+1. Set up and maintained [Gantt Chart]https://github.com/AY2021S1-CS2113-T13-1/tp/blob/master/docs/DevelopmentPractices.md#pert.
+1. Created a [Development Practices Document](https://github.com/AY2021S1-CS2113-T13-1/tp/blob/master/docs/DevelopmentPractices.md).
 
 ### Contributions to User Guide
-Created the first draft of the User Guide and provided a template for the rest to follow. Also in charge of the overall formatting for the User Guide.
-Filled up the following:
-1. Introduction
-1. Quick Start
-1. Initial draft of the features, including description and example codes.
-1. FAQ
-1. Command Summary
+Created the first draft of the [User Guide](https://github.com/AY2021S1-CS2113-T13-1/tp/blob/master/docs/UserGuide.md) and provided a template for the rest to follow. Also in charge of the overall formatting for the User Guide.
+Filled up the following(https://github.com/AY2021S1-CS2113-T13-1/tp/pull/89):
+1. [Introduction](https://github.com/AY2021S1-CS2113-T13-1/tp/pull/89)
+1. [Quick Start](https://github.com/AY2021S1-CS2113-T13-1/tp/pull/89)
+1. [Initial draft of the features](https://github.com/AY2021S1-CS2113-T13-1/tp/pull/89), including description and example codes
+1. [FAQ](https://github.com/AY2021S1-CS2113-T13-1/tp/pull/89)
+1. [Command Summary](https://github.com/AY2021S1-CS2113-T13-1/tp/pull/89)
```

Comment:
Can consider to link to the actual section of the UG

Reply:
Discussion taken out of GitHub